components/MovieDetails.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components/native';
+import { useFonts } from 'expo-font';
+import {
+  View,
+  StyleSheet,
+  StatusBar,
+  ScrollView,
+} from 'react-native';
+import { DETAILS_URL } from '../utils/urls';
+
+export const MovieDetails = ({ route, navigation }) => {
+
+  //added custom fonts
+  const [loaded] = useFonts({
+    AdventPro: require('../assets/fonts/AdventPro-SemiBold.ttf'),
+  });
```

Comment:
In order to use fonts you had to download expo-font package and use these three lines of code. What is loaded?